templates/user.yaml

```diff
@@ -13,19 +13,17 @@ users:
     admin: True
     projects:
     - auth_id: jnkns
-      privilege: [create, read, update, delete, upload, read-storage]
+      privilege: ['create', 'read', 'update', 'delete', 'upload', read-storage]
     - auth_id: jenkins
-      privilege: [create, read, update, delete, upload, read-storage]
-    projects:
+      privilege: ['create', 'read', 'update', 'delete', 'upload', read-storage]
     - auth_id: DEV
-      privilege: [create, read, update, delete, upload, read-storage]
+      privilege: ['create', 'read', 'update', 'delete', 'upload', read-storage]
     - auth_id: project1
-      privilege: [create, read, update, delete, upload, read-storage]
-    projects:
+      privilege: ['create', 'read', 'update', 'delete', 'upload', read-storage]
     - auth_id: QA
-      privilege: [create, read, update, delete, upload, read-storage]
+      privilege: ['create', 'read', 'update', 'delete', 'upload', read-storage]
     - auth_id: project1
-      privilege: [create, read, update, delete, upload, read-storage]
+      privilege: ['create', 'read', 'update', 'delete', 'upload', read-storage]
```

Comment:
Did you mean to put quotes around `read-storage` too?

Reply:
yeah, will fix that :)